### ISSUE TITLE
[Brave News]: Add OPML import and export for Brave News

### ISSUE THREAD

**Issue by mattmcalister** [brave/brave-browser#25695]
OPML support required across all platforms. This should be added in Settings in the Add Source context. Users should have the option to import OPML files where they will then automatically 'follow' all the sources from the file. Equally, users should be able to export OPML to a file that is then stored on the device. Exporting does not affect the following status of a source.

**Comment by ilovethensa** [comment]
any progress on this?

**Comment by e-motiv** [comment]
+1

**Comment by Segment0895** [comment]
I understand there is probably some uncertainty about the usefulness of this feature.
I think this can be delayed but only if Brave News sources are sync'ed.

In other words, I see two differents use cases:

Usecase 1: use Brave News for regular, slow feeds: I think a regular newseed reader is better.

Usecase 2: use Brave News for fast-paced, newspaper based, feeds: I think Brave News here is best. Thing is: I won't start adding feeds unless I'm sure they sync with my other devices.

In summary: encouraging adding feeds (by making them syncable) to Brave News may be a good way to kickstart newspapers awareness of Brave browser, micropayments (when implemented) and Brave News itself.

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using 
```
Brave | 1.94.99 Chromium: 151.0.7922.71 (Official Build) beta (64-bit)
-- | --
Revision | 7603055e3046aa01133bcbb748caa24aa39e60de
OS | Windows 11 Version 25H2 (Build 26200.8875)
```
Verified the issue using the testplan from https://github.com/brave/brave-core/pull/37710#issue-4790451914
More testing done via https://github.com/brave/brave-browser/issues/27596#issuecomment-5165743297

####  Confirmed that the 
-   Users are able to export OPML to a file  "Export to OPML" that is then stored on the device in the Brave News Dashboard 
-   Users have the option to import OPML files via  "Import from OPML"  where they will then automatically 'follow' all the sources from the file 

Light | Dark
----- | --------
<img width="1911" height="1135" alt="Image" src="https://github.com/user-attachments/assets/42f54b00-c0bb-4c03-8266-97dd4333784a" />|<img width="1913" height="1137" alt="Image" src="https://github.com/user-attachments/assets/16328f9f-d696-412c-a342-79865da0b32c" />

**Comment by hffvld** [comment]
Confirmed with @fallaciousreasoning  that this does not apply to `Android` and removed the `OS/Android` label.
